app/models/ReturnsVariation.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json.{Format, Json}
+import models.{Site, UkAddress}
+
+case class ReturnsVariation(
+                             orgName: String,
+                             ppobAddress: UkAddress,
```

Comment:
A more descriptive name please? What is ppob supposed to stand for to those who read the code in the future?

Reply:
Ramin and I reviewed the code and the back end.  ppobAddress is how the information is sent to the backend.  Will leave as is at this time.